README.md

```diff
@@ -14,30 +14,76 @@ This README contains a high level description of contributing a SWIP, for conven
  3. Add your SWIP to your fork of the repository. There is a [template SWIP here](SWIPs/swip-X.md).
  4. Submit a Pull Request to Swarm's [SWIPs repository](https://github.com/ethersphere/SWIPs).
 
-Your first PR should be a first draft of the final SWIP. It must meet the formatting criteria. An editor will manually review the 
-first PR for a new SWIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to 
-a discussion forum or open GitHub issue where people can discuss the SWIP as a whole.
+A SWIP proposal or suggested edits (or updates) may come from anyone (inside or outside the organisation) and are received in the form of a pull request to the [SWIP repository](https://github.com/ethersphere/SWIPs). Throughout the lifetime of a SWIP from proposal to final accepted, we use the PR-review infrastructure of github to comment and improve the SWIP.
 
-If your SWIP requires images, the image files should be included in a subdirectory of the `assets` folder for that SWIP as 
-follows: `assets/swip-X` (for swip **X**). When linking to an image in the SWIP, use relative links such as 
-`../assets/swip-X/image.png`.
+The github permissions of the repo reflect these privileges:
+- Anyone can open a new PR.
+- Only SWIP editors can merge a PR to master.
 
-When you believe your SWIP is mature and ready to progress past the draft phase, you should open a PR changing the state of your 
-SWIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there 
-is no rough consensus - for instance, because contributors point out significant issues with the SWIP - they may close the PR and 
-request that you fix the issues in the draft before trying again.
+### Creating a new SWIP
 
-For a detailed description of the SWIP process see [SWIP-0](SWIPs/swip-0.md).
+A new SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) titled `SWIP-<XXXX>: <title>` where `XXXX` is the SWIP number left padded with zeros up to four digits (i.e., fprinted with the format string `%04d`). 
+The PR is adding at least a new placeholder file for the new proposal named `XXXX.md` to be located in the `SWIPs` subdirectory of the SWIPs repo.
```

Comment:
Let's also define the name format here, like: swip-xx.md (where xx is the number of the PR, as discussed above)